test/e2e/apimachinery/flowcontrol.go

```diff
@@ -321,6 +321,7 @@ func createFlowSchema(f *framework.Framework, flowSchemaName string, matchingPre
 func makeRequest(f *framework.Framework, username string) *http.Response {
 	config := f.ClientConfig()
 	config.Impersonate.UserName = username
+	config.RateLimiter = clientsideflowcontrol.NewTokenBucketRateLimiter(-1, 0)
```

Comment:
Rather than hack around with numbers that do not make sense, let's just use https://github.com/kubernetes/client-go/blob/v12.0.0/util/flowcontrol/throttle.go#L103

Reply:
updated